node-zerox/src/index.ts

```diff
@@ -11,6 +11,28 @@ import fs from "fs-extra";
 import os from "os";
 import path from "path";
 import pLimit, { Limit } from "p-limit";
+import { LLMParams } from "./types";
+
+const defaultLLMParams: LLMParams = {
```

Comment:
Let's move `defaultLLMParams` and `validateLLMParams` to utils. This file should just have zerox in it.